DOC : add how to read fmriprep visual report (#21)
nipreps/nipreps.github.io: docs/users/howto_read_fmriprep_visual_report.md

# How To Read fMRIPrep visual Report

fMRIPrep generates HTML visual reports that allow you to check the results of the preprocessing steps that were applied to the data. 

> Note: this is not a replacement for running quality control of your images before running fMRIPrep (FAQ link: https://fmriprep.readthedocs.io/en/stable/faq.html). 

Reviewer: Reworking on top of Chris' suggestion:

```suggestion
Note: This is not a replacement for [QC'ing the *unprocessed* data before *fMRIPrep*](https://fmriprep.org/en/stable/faq.html#should-i-run-quality-control-of-my-images-before-running-fmriprep).
```

Below is a guide to interpreting the information contained in the visual reports, organized by subsection of the report.

If you have examples of “bad” preprocessing results that you’re willing to share, please [submit them somewhere?] with a short description of the problem (for example, “bad skull stripping”).

When you open a report, it may be helpful to first jump to the Errors subsection to see whether fMRIPrep encountered any errors during preprocessing. Note that “errors” refers only to problems in running fMRIPrep, and NOT problems with the quality of the resulting images. This section won’t flag participants in which fMRIPrep was able to successfully run to completion but yielded poor results. 

## **Summary**

> The first section of the report is a text summary, containing the subject ID, number of structural and functional images, task name, output spaces in which fMRIPrep generated your preprocessed data, and whether surface reconstruction with Freesurfer was run. 

Reviewer: ```suggestion
At the head, the report includes a textual summary of relevant aspects of the data.
These are the participant identifier, the number of structural and functional images, task(s) name(s), output spaces in which *fMRIPrep* generated your preprocessed data, and whether surface reconstruction with *FreeSurfer* was run.
```

It is good to check that all of this information is as expected.

> ## **Anatomical**

Reviewer: ```suggestion
## Anatomical section
```

The reports first present results of anatomical preprocessing. 

### Anatomical conformation

> * This section provides information about the structural images.

Reviewer: We just said this above.
```suggestion
```

* Check for obvious failures, such as missing images or implausible values for voxel size, orientation, or dimensions.

>     * For example: you’re expecting two input T1w images but it says “Input T1w images: 1”

Reviewer: Same for apostrophe:
```suggestion
    * For example: you're expecting two input T1w images but it says "Input T1w images: 1"
```

>     * For example: you’re expecting two input T1w images but it says “Input T1w images: 1”

Reviewer: Agreed.

>     * For example: you’re expecting two input T1w images but it says “Input T1w images: 1”

Author: I have addressed this comment by modifying the layout. It does not look yet very pretty but it's an improvement.
